cmd/wait-for-github/ci.go

```diff
@@ -42,31 +42,62 @@ type ciConfig struct {
 var (
 	// https://regex101.com/r/dqMmDP/1
 	commitRegexp = regexp.MustCompile(`.*github\.com/(?P<owner>[^/]+)/(?P<repo>[^/]+)/commit/(?P<commit>[abcdef\d]+)/?.*`)
+	prRegexp     = regexp.MustCompile(`.*github\.com/(?P<owner>[^/]+)/(?P<repo>[^/]+)/pull/(?P<prnumber>[\d]+)/?.*`)
```

Comment:
Can you please unify this with

https://github.com/grafana/wait-for-github/blob/03baf7d38b17e2015d6966ea2e092cbacab98f1f/cmd/wait-for-github/pr.go#L57

so we only have the regex defined once.

Probably goes along with moving `extractRefFromPrURL` and calling that from `pr.go` too.